### PR TITLE
add: p0-textutility-modal-list

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

**Review by nikhiljugale007 — COMMENTED** [review]
Code is good.
Modal component is not opening when we click on open modal button.
One suggestion in list, you can also add ordered list as well.

File: utils/typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+          <p class="comp_desc-text p-lg">
+            Text styles are, text line through
+            <span class="comp_desc_highlight">.text_line_th</span> , text
+            uppercase <span class="comp_desc_highlight">.text_uc</span> , and
+            text capitalize <span class="comp_desc_highlight">.text_cl</span>
+          </p>
+          <div class="wrapper_comp_box wrapper_typography flex">
+            <p class="text_line_th">This is a line through text.</p>
+            <p class="text_uc">This is a uppercase text.</p>
+            <p class="text_c">This is a capitalize text.</p>
```
Comment (rahulk31) [inline utils/typography/typography.html]: For capitalize you have specified the class "text_cl". Correct the typo.

File: utils/typography/typography.html [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+          <div class="comp_snippet">
+            <script type="text/plain" class="line-numbers language-markup">
+
+              <!-- typography - text styles  -->
+
+              <p class="text_line_th">This is a line through text.</p>
+              <p class="text_uc">This is a uppercase text.</p>
+              <p class="text_c">This is a capitalize text.</p>
+            </script>
```
Comment (rahulk31) [inline utils/typography/typography.html]: typo again in classname.

File: components/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  position: fixed; /* Stay in place */
+  z-index: 9999; /* Sit on top */
+  top: 0;
+  left: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
```
Comment (rahulk31) [inline components/modal/modal.css:13]: Could not preview modal. It does not opens on clicking. need to fix it.

**Review by rahulk31 — COMMENTED** [review]
Please make the suggested changes.

**Comment by gauravsinhaweb** [comment]
> Code is good. Modal component is not opening when we click on open modal button. One suggestion in list, you can also add ordered list as well.

I forgot to link modal js. Fixed the issue now!

File: components/modal/modal.css (hunk 3 above)
Comment (gauravsinhaweb) [reply components/modal/modal.css:13]: I forgot to link modal js. Fixed the issue now!

File: utils/typography/typography.html (hunk 2 above)
Comment (gauravsinhaweb) [reply utils/typography/typography.html]: fixed typo. thanks!

File: utils/typography/typography.html (hunk 1 above)
Comment (gauravsinhaweb) [reply utils/typography/typography.html]: fixed typo.

**Comment by gauravsinhaweb** [comment]
> Please make the suggested changes.

made the changes.

**Pull request merged**